mediation_service/tests/performance/common/auth.py

```diff
@@ -0,0 +1,89 @@
+import gevent.monkey
+# If gevent is installed, it needs to monkey-patch the python sockets to cooperate (see documentation or this
+# github issue https://github.com/gevent/gevent/issues/941).
+gevent.monkey.patch_all()
```

Comment:
Is this patching only necessary because this file lives in a separate directory? If that's the case let's move it one directory up. We use this pattern (flattening files) in other places as well.